projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { PluginPrePublishPanel, PluginPostPublishPanel } from '@wordpress/edit-post';
+import { createInterpolateElement, useEffect, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getSubscriberCount } from './api';
+import InspectorNotice from '../../shared/components/inspector-notice';
+import './panel.scss';
+
+export default function SubscribePanels() {
+	const [ subscriberCount, setSubscriberCount ] = useState( null );
+	useEffect( () => {
+		getSubscriberCount( count => setSubscriberCount( count ) );
+	}, [] );
+
+	return ! Number.isFinite( subscriberCount ) || subscriberCount <= 0 ? null : (
+		<>
+			<PluginPrePublishPanel
+				className="jetpack-subscribe-pre-publish-panel"
+				initialOpen
+				title={ __( 'Subscribers', 'jetpack' ) }
+			>
+				<InspectorNotice>
+					{ createInterpolateElement(
+						sprintf(
+							/* translators: %s is the number of subscribers */
+							_n(
+								'This post will be sent to <span>%s reader</span>',
+								'This post will be sent to <span>%s readers</span>',
+								subscriberCount,
+								'jetpack'
+							),
+							subscriberCount
+						),
+						{ span: <span className="jetpack-subscribe-reader-count" /> }
+					) }
+				</InspectorNotice>
+			</PluginPrePublishPanel>
+			<PluginPostPublishPanel className="jetpack-subscribe-post-publish-panel" initialOpen>
```

Comment:
Do we not want to have the "Subscribers" title in there, just like before we hit publish?

Reply:
We'd actually like to move this inspector notice to be directly adjacent to the site card component as per the design specs, but this was the best we could do at this time. It might be worth revisiting in the future, however.

p1HpG7-etk-p2#the-publish-flow